### DISCUSSION TITLE
Create custom app

### DISCUSSION THREAD

**Discussion by elmundomusical2** [WhatsApp/stickers#791] [Q&A]
Hi! I want to integrate stickers on my existing app. Is there any tutorial for do it? I have try for weeks but it doesn't work anything....
Thanks! 🙏🙏

**Comment by simonzhexu** [comment]
You will need to implement https://github.com/WhatsApp/stickers/tree/master/Android#advanced-development for Android.  https://github.com/WhatsApp/stickers/tree/master/iOS#api-documentation for iOS.
